frontend/src/components/WorkspaceSettings/components/Members/components/InviteMember.tsx

```diff
@@ -48,18 +48,21 @@ export const InviteMember = ({ open, closeDialog }: MembersActionDialog) => {
 
   const handleInviteMember = async () => {
     const res = await inviteUser({ data: { email: convertEmailsIntoAnArray(email) } });
-    setErr((res as resError)?.error_message ?? 'none');
-  };
 
-  useEffect(() => {
-    if (err === 'none') {
+    if ((res as resError)?.error_message) {
+      const errorMessage = (res as resError).additional_information?.errors?.map(e => e.msg).join(', ');
```

Comment:
even if the format has changed, it's logic  that should be handled on the server, error_message should includes all the info user needed.

Reply:
It includes but not in the way we need it, I have FE ticket for this.
But you are right, this kind of things
<img width="475" alt="Screenshot 2023-06-01 at 14 35 11" src="https://github.com/deepchecks/monitoring/assets/42544127/a5d051cd-230c-4b3a-9418-46491894eb93">
 should be handled on the BE.

Reply:
I commented on the relevant linear issue